### ISSUE TITLE
Allow keyboard shortcuts for Custom Actions

### ISSUE THREAD

**Issue by mannodermaus** [ForkIssues/Tracker#438]
I'd like to assign a keyboard shortcut to a Custom Action added to Fork.

**Comment by DanPristupov** [comment]
That is a good request.

Shortcuts can also be useful for upcoming file custom actions (https://github.com/ForkIssues/Tracker/issues/417)

**Comment by 153957** [comment]
This is already possible via the macOS shortcuts system.

Open System Preferences -> Keyboard -> Shortcuts -> App Shortcuts -> +
Then choose Application: `Fork.app`, Menu Title: [name of your action], and then choose your shortcut.

**Comment by mannodermaus** [comment]
@153957, thanks for the info! That does indeed work.

**Comment by SassNinja** [comment]
@DanPristupov any update on this?

It would be a huge improvement if I could execute my custom commands with custom shortcuts instead of doing an endless amount of clicks.

**Comment by joshgold22** [comment]
> This is already possible via the macOS shortcuts system.
> 
> Open System Preferences -> Keyboard -> Shortcuts -> App Shortcuts -> +
> Then choose Application: `Fork.app`, Menu Title: [name of your action], and then choose your shortcut.

This only works for anything in the standard Mac menubar.   For actions listed only in the context-menu (is this only "File" custom actions,) I don't think there is a way to create a shortcut.

**Comment by certainlyakey** [comment]
@joshgold22 true. It's not possible to open a selected file in an editor via shortcut, since custom commands on a file do not make it to the menubar. 
Eg. if i create a custom command to open a file in VS Code: 
![image](https://user-images.githubusercontent.com/1833029/143227182-38bf8a0b-cdac-4232-8b65-cbc71ccef51f.png)
the command doesn't appear in any of the existing menus (File, Edit, View, Repository, Window) and therefore you need to go through a sequence of clicks and mouse movements to open it.

**Comment by jasonfharris** [comment]
It would be great if a custom keystroke was added for these custom commands. (I have custom commit commands.)
